### PR TITLE
Annie/navbar

### PR DESCRIPTION
**Pull request by anniebelkin**

### CODE REVIEW

File: src/components/Button/Button.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import './Button.scss';
+
+const Button = props => {
+
+    return (
+        <button 
+            type={ props.type || 'button' } 
+            className={ 'Button ' + (props.className || '') }
```
Comment (ozizhakGT) [inline src/components/Button/Button.js]: consider user className={`Button ${props.className}`}

File: src/components/Button/Button.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import './Button.scss';
+
+const Button = props => {
+
+    return (
+        <button 
+            type={ props.type || 'button' }
```
Comment (ozizhakGT) [inline src/components/Button/Button.js]: consider use `` es6 .. then you dont need to use | |

File: src/components/Navbar/Navbar.js [hunk 3]
```diff
@@ -1,17 +1,40 @@
 import React from "react";
-import './Navbar.scss';
 import {Link} from 'react-router-dom';
+import SearchBar from '../SearchBar/SearchBar.js';
+import Button from '../Button/Button.js';
+import Logo from '../Logo/Logo';
+import './Navbar.scss';
```
Comment (ozizhakGT) [inline src/components/Navbar/Navbar.js]: organize your imports.
.scss navbar file meed to be after React import
js file play synchrony

File: src/components/Navbar/Navbar.js [hunk 4]
```diff
@@ -1,17 +1,40 @@
...
 const Navbar = props => {
-    console.log(props);
+
+    const isConnected = props.isConnected || false;
+    let accountPanel = '';
+
+    if (isConnected) {
+        accountPanel = <div className='Navbar__account'>
+                        <Button value={ 'צור מפגש' } className={ 'Button__black' } />
+                        <Button value={ 'א' } className={ 'Button__white Button__circle Navbar__account--profile' } />
```
Comment (ozizhakGT) [inline src/components/Navbar/Navbar.js]: value from props !
try to do it  as real as possible

**Review by ozizhakGT — COMMENTED** [review]
see note @anniebelkin

**Review by ozizhakGT — COMMENTED** [review]
Great Job!

**Comment by ozizhakGT** [comment]
GJ ! :)

**Review by Avishayy — APPROVED** [review]
Nice!

**Pull request merged**